OpenRA.Mods.Common/Widgets/Logic/AssetBrowserLogic.cs

```diff
@@ -217,6 +218,16 @@ public AssetBrowserLogic(Widget widget, Action onExit, World world, Dictionary<s
 			else
 				allowedExtensions = new string[0];
 
+			foreach (var package in Game.ModData.ModFiles.MountedPackages)
```

Comment:
You may be able to simplify this to something like `var acceptablePackages = Game.ModData.ModFiles.MountedPackages.Where(p => p.Contents.Any(c => allowedExtensions.Contains(Path.GetExtension(c))));`